### ISSUE TITLE
ClickOnce fails in Chrome

### ISSUE THREAD

**Issue by yodurr** [chocolatey/boxstarter#345]
Running the clickonce installer fails to start in Chrome.  It's a known issue and the workaround is to use Edge.  I didn't see an issue for it so adding the issue here.

**Comment by gep13** [comment]
@yodurr thanks for raising this.

This has been a known issue for a while, and it comes down to the fact that Chrome removed support for NPAPI:

https://blog.chromium.org/2014/11/the-final-countdown-for-npapi.html

It is my understanding that ClickOnce relies on this in order to function, and as a result, fails.  As a result, I don't think that there is much that Boxstarter itself, can do to work around this.

@Redsandro has created a Google Chrome extension (https://chrome.google.com/webstore/detail/windows-remix-clickonce-h/dgpgholdldjjbcmpeckiephjigdpikan) which adds support for ClickOnce back into those browsers, however, this would need to be installed, prior to trying to access a Boxstarter URL.

@yodurr did you have any suggestions on how to workaround this in another way?

**Comment by pauby** [comment]
Issues #61 and #97 both talk about the FF and Chrome issues around this.

**Comment by pauby** [comment]
@yodurr Did you manage to get this working in Chrome?

**Comment by yodurr** [comment]
no, it still doesn't work.  Am using the scripts on aka.ms/devsetup

**Comment by pauby** [comment]
As far as I understand it this is a [browser issue](https://github.com/chocolatey/boxstarter/issues/345#issuecomment-422058923) and outside of Boxstarter. 

I was hoping you were going to come back with a 'yeah, and here is a fantastic plugin that works really well' ... :(

**Comment by AnthonyMastrean** [comment]
At some point, the IE will be dead and the Edge that runs with NPAPI will also be dead, replaced by the Chromium Edge. https://www.windowscentral.com/microsoft-building-chromium-powered-web-browser-windows-10
